packages/frontend/amp/components/elements/ImageBlockComponent.tsx

```diff
@@ -4,6 +4,10 @@ import { sans } from '@guardian/pasteup/fonts';
 import { palette } from '@guardian/pasteup/palette';
 import { css } from 'react-emotion';
 import { pillarPalette } from '@frontend/lib/pillars';
+import {
+    getImage,
+    getImageHeight,
+} from '@root/packages/frontend/amp/components/lib/GetImage';
```

Comment:
Would `@frontend/amp/components/lib/GetImage` work here?

Reply:
i will check

Reply:
it does!

Reply:
Lovely stuff! Thanks